otx/algorithms/action/adapters/mmaction/task.py

```diff
@@ -117,6 +117,9 @@ def _init_task(self, export: bool = False):  # noqa
             )
         self._recipe_cfg.log_config.hooks.append({"type": "OTXLoggerHook", "curves": self._learning_curves})
 
+        if self._hyperparams.learning_parameters.auto_num_workers:
```

Comment:
Could this part move to patch_from_hyperparameters function?

Reply:
Actually, I tried to, but code `self._update_caching_modules(self._recipe_cfg.data)` located just below uses `num_workers` value. So, code should be located before code for caching.

Reply:
patch_from_hyperparams is already located before _update_caching_modules. I can't understand your point

Reply:
Oh, I misunderstood as it's in `self.configure`. I'll update it.

Reply:
moved it.